corehq/blobs/migrate_metadata.py

```diff
@@ -92,13 +92,19 @@ def _do_migration(self, doc):
             type_code = self.get_type_code(doc)
             obj = self.blob_helper(doc, self.couchdb, type_code)
             db = get_db_alias_for_partitioned_doc(doc["_id"])
```

Comment:
Changed this in the rebase. It had mistakenly been changed to `obj._id`, which was wrong.